### PR TITLE
feat(hostinfo/packages): Added functionality to check if auto updates…

### PR DESCRIPTION
**Pull request by kaustavha**
… are enabled; required by waldo.

### CODE REVIEW

File: hostinfo/packages.lua [hunk 1]
```diff
@@ -30,47 +31,100 @@ end
...
+          end
+        end
+      end
+      return cb()
+    end
+    execFileToBuffers(pkgcmd, pkgargs, options, statExecCb)
+
+  end
+  }, function(err)
+    table.insert(errors, err)
```
Comment (rphillips) [inline hostinfo/packages.lua]: should this only be added to errors if err is not nil?

File: hostinfo/packages.lua [hunk 2]
```diff
@@ -30,47 +31,100 @@ end
...
+    if err and not next(autoupdates) and not next(packages) then
+      self._errors = table.concat(errors, ' ')
+      return callback()
+    end
+    if update_method ~= nil and update_method ~= '' and status ~= '' and status ~= nil then
```
Comment (rphillips) [inline hostinfo/packages.lua]: ```if not update_method and #update_method > 0 and status and #status > 0```

File: hostinfo/packages.lua [hunk 3]
```diff
@@ -30,47 +31,100 @@ end
...
+      table.insert(autoupdates, {
+        update_method = method,
+        status = status
+      })
+    end
+    if autoupdates == nil or not next(autoupdates) then autoupdates = 'No information found' end
+    if packages == nil or not next(packages) then packages = 'No information found' end
```
Comment (rphillips) [inline hostinfo/packages.lua]: should make 'No information found' a constant.

File: hostinfo/packages.lua (hunk 2 above)
Comment (kaustavha) [inline hostinfo/packages.lua]: if update_method and #update_method > 0 and status and #status > 0 
Changes, think i did that in a couple other places as well so fixed those too

**Comment by rphillips** [comment]
+1

**Comment by rphillips** [comment]
this branch has conflicts

**Pull request merged**